client/src/app/primitives/modal/Modal.js

```diff
@@ -59,6 +59,7 @@ export default class Modal extends PureComponent {
     return ReactDOM.createPortal(
       <div className={ css.ModalOverlay } onClick={ this.handleBackgroundClick }>
         <div className={ classNames(css.ModalContainer, className) } ref={ this.modalRef }>
+          <Close onClick={ this.close } />
```

Comment:
If we go for this, let's not remove the export for the CLOSE button.

Reply:
Sure.